java/ql/test/library-tests/dataflow/capture/inlinetest.expected

```diff
@@ -171,10 +171,12 @@ edges
 | B.java:154:17:154:28 | source(...) : String | B.java:175:5:175:6 | String s2 : String | provenance |  |
 | B.java:158:7:158:13 | parameter this : MyLocal [String s1] : String | B.java:159:18:159:19 | this : MyLocal [String s1] : String | provenance |  |
 | B.java:158:7:158:13 | parameter this : MyLocal [String s2] : String | B.java:160:14:160:15 | this : MyLocal [String s2] : String | provenance |  |
+| B.java:158:7:158:13 | parameter this : MyLocal [String s2] : String | B.java:160:14:160:15 | this : MyLocal [String s2] : String | provenance |  |
```

Comment:
Are these test changes expected?

Reply:
Yes. When a callable has both flow-through and flow to a sink without flow-through and the difference is dependent on the access path (and not just if it is empty or not), then we generate slightly more nodes, since we can no longer determine with the same level of precision whether we need to remember a summary context or not. To pinpoint it in the code, this is due to the change to `PrevStage::parameterMayFlowThrough` using an `emptyAp` boolean instead of `ApApprox` in its use in `callEdgeArgParamRestricted`.

Reply:
Good to know that this is expected.